### ISSUE TITLE
[BUG] #148 still an issue

### ISSUE THREAD

**Issue by working-name** [linuxserver/docker-deluge#196]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Current Behavior

https://github.com/linuxserver/docker-deluge/issues/148

Deluge crashes when adding multiple magnet links or consecutive magnet links, and/or .torrent files.

### Expected Behavior

No crashes should happen.

### Steps To Reproduce

1. run container
2. add more than 1 magnet links or more than 1 torrent files OR add 1 at a time, sequentially, fairly fast.

### Environment

```markdown
- OS: debian 11
- How docker service was installed: 24.0.7, with compose plugin 2.21.0
```

### CPU architecture

x86-64

### Docker creation

```bash
the vanilla docker-compose stuff.
```

### Container logs

```bash
same as the person who reported it
```

**Comment by working-name** [comment]
Nothing more frustrating than trying to help only to be met with silly non-replies, non-communication. Great going!

**Comment by j0nnymoe** [comment]
Your issue was marked as invalid due to not completely the issue template.

**Comment by working-name** [comment]
> A human has marked this issue as invalid, this likely happened because the issue template was not used in the creation of the issue.

What part of the template is missing?

**Comment by Roxedus** [comment]
this. 
![image](https://github.com/linuxserver/docker-deluge/assets/7110194/1690ba83-dcbd-44b8-b1ee-3e6135fd08ab)

**Comment by working-name** [comment]
First:
Click the mentioned other bug report - #148, look at the log. 

Second:
Vanilla means I didn't change anything in the docker-compose from what you distribute.

**Comment by aptalca** [comment]
Honestly, if you can't be bothered to answer the few questions we ask, we can't be bothered to look into your issue. Thanks for reaching out. Feel free to open a new issue if decide to comply with our requests.

**Comment by working-name** [comment]
No, I get that 100%. I do issue triage too. I'm just not looking for excuses not to consider reports. If your time is as precious as you'd like me to believe by throwing this hissy fit, you would have made that 1 extra click required to "complete" your view on the issue at hand.

Incredible how poor your attitude is, as a group. 3 people replied, and all 3 in the same vein...

**Comment by working-name** [comment]
Well, I'm sorry, but I have no idea why or what exactly triggers it to crash and the log is as succinct as you saw. The behavior I saw consistently around crashes is when I add multiple things at once (or sequentially) to download. `.torrent` or `magtnet` - doesn't matter.

I don't know how to get you more info to troubleshoot or reproduce.

In fact, given the logs, I've wondered if it's deluge itself crashing because of a bug, or libtorrent causing it to. I've already filed a bug with deluge a while back but nothing became of it.

So if this isn't enough, I get it, and it's cool. Thanks for looking into it.
